tests/Integration/FuzzySearchTest.php

```diff
@@ -52,6 +52,7 @@ public function test_models_can_be_found_using_field_and_value_and_transposition
             ->field('title')
             ->value('boko')
             ->transpositions(true)
+            ->sort('author_id')
```

Comment:
One question - why do we sort by `author_id` everywhere? Wouldn't it be better to sort by `_id`? In theory, different books might have the same author and then the order might be not as expected.

Reply:
_id makes more sense, but maybe we can add this https://gist.github.com/pokap/ac5fce3570c9bec3a87a8908bcbd0bbc to fix the tests without sprinkling sort everywhere. What do.u think?